lib/buildTriangleStaticUniformGrid.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+var Cesium = require('cesium');
+var AxisAlignedBoundingBox = Cesium.AxisAlignedBoundingBox;
+var Cartesian3 = Cesium.Cartesian3;
+var StaticUniformGrid = require('./StaticUniformGrid');
+var triangleAxisAlignedBoundingBoxOverlap = require('./triangleAxisAlignedBoundingBoxOverlap');
+
+module.exports = buildTriangleStaticUniformGrid;
+
+/**
+ * Function for generating a uniform grid from a list of triangles of form [Cartesian3, Cartesian3, Cartesian3]
+ *
+ * @param {List[Cartesian3, Cartesian3, Cartesian3]} [data] Triangle data to bin.
+ * @param {Number} [cellWidth] The width of each cell in the uniform grid.
+ *
+ * @returns a StaticUniformGrid storing triangles.
+ */
+
+function buildTriangleStaticUniformGrid(data, cellWidth) {
```

Comment:
Replace this with a function like `StaticUniformGrid.fromTriangleSoup`.

See https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#from-constructors